pkg/cache/pkgs.go

```diff
@@ -34,6 +35,7 @@ func (pkgs packageIndex) GetPackage(_ context.Context, name string) (*registry.P
 			CurrentCSVName: ch.Head,
 		})
 	}
+	sort.Slice(channels, func(i, j int) bool { return strings.Compare(channels[i].Name, channels[j].Name) < 0 })
```

Comment:
This come up in a different conversation somewhere didn't it? Did we never fix this?

Reply:
Maybe you're thinking of [operator-lifecycle-manager/pull/2925](https://github.com/operator-framework/operator-lifecycle-manager/pull/2925) ?